packages/react-native/React/Fabric/Mounting/ComponentViews/View/RCTViewComponentView.mm

```diff
@@ -432,6 +432,17 @@ - (BOOL)canBecomeFocused
 // This is only done once the view is focused.
 //
 - (void)enableDirectionalFocusGuides
+{
+  [self enableNextFocusUp];
+
+  [self enableNextFocusDown];
+
+  [self enableNextFocusLeft];
+
+  [self enableNextFocusRight];
```

Comment:
I'd personally delegate the responsibility of checking if the view is focused or not to this function. Effectively making it defensive. It'd protect us from accidentally calling it without the check in some other places

Reply:
Agreed